crewai_tools/tools/arxiv_paper_tool/arxiv_paper_tool.py

```diff
@@ -0,0 +1,137 @@
+import os
+import re
+import time
+import urllib.request
+import urllib.parse
+import urllib.error
+import xml.etree.ElementTree as ET
+from typing import Type, List, Optional
+from pydantic import BaseModel, Field
+from crewai.tools import BaseTool
+from typing import ClassVar
+import logging
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
drop the `logging.basicConfig(level=logging.INFO)`. You can keep the `logger = logging..` 

```suggestion
logger = logging.getLogger(__name__)
```

We recently discovered a bug where excessive logs appeared during Crew execution because of this setup. Setting logging.basicConfig(level=...) overrides the logging configuration for all loggers, including those from your Crews, flooding your execution with unwanted logs.

Reply:
updated